lib/driverProviders/driverProvider.js

```diff
@@ -34,6 +34,7 @@ DriverProvider.prototype.getExistingDrivers = function() {
 DriverProvider.prototype.getNewDriver = function() {
   var builder = new webdriver.Builder().
     usingServer(this.config_.seleniumAddress).
+    usingWebDriverProxy(this.config_.seleniumProxy).
```

Comment:
Could we name it `webDriverProxy`? Unless there's a reason to have a discrepancy between the function name and the config name.